Tasks/Common/TlsHelper_/TlsHelper_.psm1

```diff
@@ -0,0 +1,51 @@
+Import-VstsLocStrings -LiteralPath $PSScriptRoot/module.json
+
+function Add-Tls12InSession {
+    [CmdletBinding()]
+    param()
+
+    try {
+        if ([Net.ServicePointManager]::SecurityProtocol -notcontains 'Tls12') {
+            [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]3072
+            Write-Verbose 'Added TLS 1.2 in session.'
```

Comment:
Should come from resx

Reply:
others as well

Reply:
I don't think all strings need to be added to module.json. Many tasks are directly writing log statements without necessarily localizing them. These are anyways only for developer debugging and don't provide much useful info to user. So I guess it should be ok.

Reply:
This statement should not be debug. You need to inform user that TLS1.2 is being set